hescorehpxml/schemas/hescore_json.schema.json

```
@@ -365,6 +365,11 @@
                                                 "$id": "#/properties/building/properties/zone/properties/zone_roof/items/properties/zone_skylight/properties/solar_screen",
                                                 "type": "boolean",
                                                 "description": "Does this window have a solar screen?"
+                                            },
+                                            "storm_window_type": {
+                                                "type": "string",
+                                                "enum": ["clear", "low-e"],
+                                                "description": "Type of storm window"
```

It's a little weird to have window in the name but it applies to a skylight. How about:

```suggestion
                                            "storm_type": {
                                                "type": "string",
                                                "enum": ["clear", "low-e"],
                                                "description": "Type of storm for this skylight"
```